libs/remix-ui/search/src/lib/context/context.tsx

```diff
@@ -208,11 +208,11 @@ export const SearchProvider = ({
         const files = await getDirectory('/', plugin)
         const pathFilter: any = {}
         if (state.include){
-          const includeWithGlobalExpression = state.include.replaceAll(/(?<!\/)(\*\.)/g, '**/*.')
+          const includeWithGlobalExpression = state.include.replaceAll(/(\*\.)/g, '**/*.')
```

Comment:
Wouldn't this change affect the correctness of the matching pattern?

Reply:
I've changed it